tests/pathfinding/test_capacity_updates.py

```diff
@@ -0,0 +1,506 @@
+"""
+The tests in this module mock events creation by using a mock blockchain listener.
+
+The Capacity Updates should be tested in various states
```

Comment:
That doesn't add so much information.

Reply:
reformulated into 
`The Capacity Updates show different correct and incorrect values to test all edge cases`